Examples/DemoPlacementViewController.swift

```diff
@@ -99,24 +102,18 @@ class DemoPlacementViewController: UIViewController {
 
-        for latlon in latlons {
+        ///Create an array of CLLocation objects to generate a polyline
+        var locationArray: [CLLocation] = []
+        // change the stride if you want to use fewer of the lat/lon pairs
```

Comment:
What does a stride unit represent?

Reply:
Worth commenting. Stride represents how many points to move by, so a stride of 1 means use every point, 2 means use every second point, etc.